t/09_qesapdeployment.t

```diff
@@ -945,38 +921,6 @@ subtest '[qesap_prepare_env] die for missing argument' => sub {
     dies_ok { qesap_prepare_env(); } "Expected die if called without provider arguments";
 };
 
-subtest '[qesap_prepare_env] integration test' => sub {
```

Comment:
Why to delete this test? I think it could be still relevant

Reply:
ok, I'll look into this.  I made the change because of `set -o pipefail` is no longer in the middle of the command.